custom/icds_reports/static/js/directives/indie-map/indie-map.directive.js

```diff
@@ -163,6 +163,20 @@ function IndieMapController($scope, $compile, $location, $filter, storageService
                     projection.scale(scaleCenter.scale)
                         .center(scaleCenter.center)
                         .translate([element.offsetWidth / 2, element.offsetHeight / div]);
+
+                    //setting zoom out limit
+                    //references: https://data-map-d3.readthedocs.io/en/latest/steps/step_06.html#step-06
+                    $(function(){
+                        // DOM Ready
+                        var svg = d3.select('#map svg'); //selects svg in datamap component
+                        var zoom = d3.behavior.zoom().scaleExtent([1, 10]).on('zoom', function () {
```

Comment:
Where does `[1, 10]` come from?

Reply:
[1, 10] is the zoom range. 1 being least the map can be zoomed out and 10 as max